lib/WelcomePage/signInButtons.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
+
+class BottomSquareWithButton extends StatelessWidget {
```

Comment:
Thanks for your contribution! I was wondering if there's a better name for this widget. For example, it might be better to call it `ButtonContainer`.